sdk/servicebus/service-bus/changelog.md

```diff
@@ -1,11 +1,17 @@
+# 2019-09-23 1.1.0
+
+- Update library to support running all Service Bus operations in browser.
```

Comment:
Add browser support. Authentication with `@azure/identity` is not yet supported &ndash; use a connection string instead.

Reply:
Updated